### PR TITLE
New pacakge: MiauLightouch.obs-virtual-cam 2.1.2

### PR DESCRIPTION
**Pull request by shanto**
This is going to replace Fenrirthviti.obs-virtual-cam which is no longer maintained. See: https://obsproject.com/forum/resources/obs-virtualcam.949/

New maintainer's post here: https://obsproject.com/forum/resources/obs-virtualcam.1744/

Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/172257)

### CODE REVIEW

**Comment by shanto** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Hi @shanto,

We have a limit of 1 PR change (add/modify/remove) per PR. Unfortunately, moving a package to a different `PackageIdentifier` is seen by the system as 2 changes: remove from one `PackageIdentifier` and add to another. If either of these could be moved to another PR, then both should proceed past this check.

- The `.validation` file is used by our automation to record when a waiver has been applied to a package. I'm not exactly sure how it will be handled - the system might automatically clean it up, or we might need to do so manually. But it will also block, as the system expects PRs for the `\manifests` folder to only have `.yaml` files.

**Comment by shanto** [comment]
> Hi @shanto,
> 
> We have a limit of 1 PR change (add/modify/remove) per PR. Unfortunately, moving a package to a different `PackageIdentifier` is seen by the system as 2 changes: remove from one `PackageIdentifier` and add to another. If either of these could be moved to another PR, then both should proceed past this check.
> 
> * The `.validation` file is used by our automation to record when a waiver has been applied to a package. I'm not exactly sure how it will be handled - the system might automatically clean it up, or we might need to do so manually. But it will also block, as the system expects PRs for the `\manifests` folder to only have `.yaml` files.

Thanks @stephengillie for the explanation. I have reverted the cleanup part. Hopefully it will go through.

I will submit another PR later to cover the deletion. Is there any _replace/provides_ mechanism in Winget as seen in Linux package managers? Just curious.

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 895.)

**Comment by stephengillie** [comment]
> Is there any replace/provides mechanism in Winget as seen in Linux package managers? Just curious.

There *is* an advanced type of PR that adds one package version and removes another package version in the same `PackageIdentifier`, and I frequently approve these. But I am only of about intermediate GIt experience and do not know how to create this type of PR.

image

Is an installer switch needed to bypass this and continue the install silently?

File: manifests/m/MiauLightouch/obs-virtual-cam/2.1.2/MiauLightouch.obs-virtual-cam.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: MiauLightouch.obs-virtual-cam
+PackageVersion: 2.1.2
+InstallerType: nullsoft
+InstallerSwitches:
+  Silent: '/VERYSILENT /SUPPRESSMSGBOXES /CLOSEAPPLICATIONS'
+  SilentWithProgress: '/VERYSILENT /SUPPRESSMSGBOXES /CLOSEAPPLICATIONS'
+Installers:
```
Comment (stephengillie) [inline manifests/m/MiauLightouch/obs-virtual-cam/2.1.2/MiauLightouch.obs-virtual-cam.installer.yaml:9]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: OBSProject.OBSStudio
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
This package somehow installed onto the Desktop. It appears to be an OBS Studio plugin. Should `OBSProject.OBSStudio` be a dependency?
image

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
Verify:
image
I'm guessing that this is from the plugin, since the name is similar. 
image

**Pull request merged**